### DISCUSSION TITLE
Add option to extend OTP for visually impaired peple.

### DISCUSSION THREAD

**Discussion by cipineda** [Authenticator-Extension/Authenticator#1176] [Ideas]
We use this Authenticator and works great, thanks for creating this.
We do have a request though. Is it possible to add an option to extend the duration of the OTP?
We have people that are visually impaired or have other type of restriction based on ADAAA.
They are slower to copy/paste, switch screens, etc in order to complete the MFA authentication, and 30 seconds does is not enough for them.
Your help is greatly appreciated.

**Comment by mymindstorm** [comment]
Unfortunately, the 30s validity window is part of the TOTP specification and set by the account provider ([[1]](https://datatracker.ietf.org/doc/html/rfc6238#section-5.2), [[2]](https://github.com/google/google-authenticator/wiki/Key-Uri-Format#period)). I would recommend enabling the autofill feature in settings or seeing if it is viable to switch to passkeys/yubikeys.

**Comment by cipineda** [comment]
Thanks for the prompt response.
Cesar
